src/dashboard/src/main/migrations/0008_fpcommandoutput.py

```diff
@@ -0,0 +1,18 @@
+# -*- coding: utf-8 -*-
+from __future__ import unicode_literals
+
+from django.db import models, migrations
+
+
+class Migration(migrations.Migration):
+
+    dependencies = [
+        ('main', '0007_django_upgrade_tweaks'),
+    ]
+
+    operations = [
+        migrations.AlterModelTable(
+            name='fpcommandoutput',
+            table='FPCommandOutput',
+        ),
```

Comment:
This needs to be updated to handle legacy databases.

Context: We originally had a table named FPCommandOutput.  When we added models, a FPCommandOutput model was created, but no legacy table name was created, so Django created main_fpcommandoutput.  This was later fixed (ab97a22203031bfca8e2bfa8906b18e9a62c2857) and this migration is intended to move main_fpcommandoutput to FPCommandOutput, but it fails if the legacy FPCommandOutput already exists.  We need to establish which table actually contains user data, and preserve that when we merged the two tables.